janusgraph-examples/README.md

```diff
@@ -17,23 +17,110 @@ the various supported storage and indexing backends.
 
 ## Prerequisites
 
-* Java 8 Developer Kit, update 40 or higher
-* Apache Maven, version 3.3 or higher
+* Java 8 Development Kit, update 40 or higher. [Oracle Java](https://www.oracle.com/java/index.html) and [OpenJDK](http://openjdk.java.net/) have been tested successfully.
+* [Apache Maven](http://maven.apache.org/), version 3.3 or higher
 
-## Building the Examples
+### JanusGraph distribution
+
+Download the latest [JanusGraph release](https://github.com/JanusGraph/janusgraph/releases)
+distribution zip file and unzip it.
+
+### Building the examples
+
+The examples are packaged with the JanusGraph distribution zip file in the
+`examples` directory. Run the Maven build command from the `examples` directory.
 
 ```
 mvn clean install
 ```
 
-## Running the Examples
+## Running the examples
+
+These commands are run from the `examples` directory, but please refer to
+the specific directions for each project as there may be additional setup
+and configuration required.
+
+### [Common and In-Memory](example-common/README.md)
+
+```
+mvn exec:java -pl example-common
+```
+
+### [BerkeleyJE and Lucene](example-berkeleyje/README.md)
+
+```
+mvn exec:java -pl example-berkeleyje
+```
+
+### [Cassandra Thrift and Elasticsearch](example-cassandra/README.md)
+
+```
+mvn exec:java -pl example-cassandra
+```
+
+### [Cassandra CQL and Elasticsearch](example-cql/README.md)
+
+```
+mvn exec:java -pl example-cql
+```
+
+### [HBase and SolrCloud](example-hbase/README.md)
+
+```
+mvn exec:java -pl example-hbase
+```
+
+### [HBase and Solr Standalone (HTTP)](example-hbase/README.md#hbase-and-solr-standalone-http)
+
+```
+mvn exec:java -pl example-hbase -Dexample.config="example-hbase/conf/jgex-hbase-solr-http.properties"
```

Comment:
Use `mvn exec:java -pl :example-hbase -Dexample.config="\${project.basedir}/conf/jgex-hbase-solr-http.properties"` so it can be run anywhere in the project.